packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1759,7 +1760,14 @@ export default class Meeting extends StatelessWebexPlugin {
         if (err.meetingInfo) {
           this.meetingInfo = err.meetingInfo;
         }
+        throw new PermissionError();
+      } else if (err instanceof MeetingInfoV2WebinarRegistrationError) {
+        this.meetingInfoFailureReason = MEETING_INFO_FAILURE_REASON.WEBINAR_REGISTRATION;
+        this.meetingInfoFailureCode = err.wbxAppApiCode;
 
+        if (err.meetingInfo) {
+          this.meetingInfo = err.meetingInfo;
+        }
         throw new PermissionError();
```

Comment:
just to double check: we're still throwing the same `PermissionError` here as for `MeetingInfoV2PolicyError` - is this correct?

Reply:
Yes, i create a new JoinStatusError for this case. Current handle the error code is belong registration.